pkg/reconciler/kubernetes/tektonresult/transform.go

```diff
@@ -82,6 +84,18 @@ func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp
 	return common.Transform(ctx, manifest, instance, extra...)
 }
 
+func isDBExternal(p v1alpha1.ResultsAPIProperties) mf.Transformer {
```

Comment:
the name of the function say, it returns `bool`, can you please rename it?

Reply:
Yes, agreed.

Reply:
Changed to filterExternalDB.